lambdas/nino-check/src/hmrc-apis/pdv.ts

```diff
@@ -0,0 +1,67 @@
+import { PdvApiErrorBody, PdvApiInput, PdvApiResponseBody, PdvConfig, PdvFunctionOutput } from "./types/pdv";
+import { logger } from "../../../common/src/util/logger";
+import { captureLatency } from "../../../common/src/util/metrics";
+import { safeStringifyError } from "../../../common/src/util/stringify-error";
+
+export async function matchUserDetailsWithPdv(
+  { apiUrl, userAgent }: PdvConfig,
+  oAuthToken: string,
+  apiInput: PdvApiInput
+): Promise<PdvFunctionOutput> {
+  const [response, latency] = await captureLatency("MatchingHandler", () =>
```

Comment:
Annoying that we're now tied to these metric names even though they don't make sense any more!